Automattic/wp-calypso
# Gutenboarding: Intent Capture update

#### Changes proposed in this Pull Request
* Remove vertical select from Intent Capture step in Gutenboarding.
* Update site title input styling and use a regular `<input/>` instead of `contenteditable`.
* Design Selection step (`/new/design` route) can be now accessed with an empty state.

#### Testing instructions
* Go to [/new](https://calypso.live/new?branch=update/gutenboarding-intent-capture-remove-vertical).
* Advancing from Intent Capture should work by:
  * typing at least 3 characters and pressing the primary button _Choose Design_
  * pressing _skip for now_ link below the input. in this case any input will be removed and the header will reflect default state with DomainPicker using the fallback search query _My new site_. 

Fixes #42363
